net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/BGP.xml

```diff
@@ -364,6 +364,7 @@
                                                 <source>OPNsense.quagga.bgp</source>
                                                 <items>prefixlists.prefixlist</items>
                                                 <display>name</display>
+                                                <group>name</group>
```

Comment:
In set this to grouping because without it will list every single entry of a list. But every entry belongs to the same list having the same name, so it makes sense to just list lists by list name and not by every entry made.
I can provide Screenshots of the behaviors before and after, and the configs generated.

Reply:
the name is the actual key here, missed that part. A bit confusing in my opinion, but just the way it works.